### ISSUE TITLE
First Release Requirements (0.1.0)

### ISSUE THREAD

**Issue by RobDavenport** [gamercade-io/gamercade_console#54]
This is a tracking issue for any tasks which need to be completed prior to announcing our first real release. If theres any other tasks you think might be important or critical to it, please reply to this issue and make suggestions.

The "Nice To Haves" List I can see as being priority issues in the near future/future releases.

**Hard Requirements:**
- Site Documentation
  - [x] https://github.com/gamercade-io/gamercade_site/issues/3
  - [x] https://github.com/gamercade-io/gamercade_site/issues/4
  - [x] ~~https://github.com/gamercade-io/gamercade_site/issues/5~~
  - [x] https://github.com/gamercade-io/gamercade_site/issues/10
  - [x] https://github.com/gamercade-io/gamercade_site/issues/11
  - [x] https://github.com/gamercade-io/gamercade_site/issues/14
  - [x] https://github.com/gamercade-io/gamercade_site/issues/15
- [x] #24 
- [x] #50
- [x] #31 
- [x] #13 
- [x] #55 
- [x] #56 
- [x] #10 
- [x] #57 
- [x] #60 
- [x] #64 
- [x] #72 
- [x] #74 
- [x] #76 
- [x] #2 
- [x] #6 
- [x] #77 
- [x] #3 
- [x] #4 
- [x] #63 
- [x] https://github.com/gamercade-io/gamercade_site/issues/8
- [x] #8 
- [x] #42 
- [ ] #61 
- [x] #22
- [x] #68 

**Nice To Haves:**
- [ ] #62 
- [ ] #9 
- [ ] https://github.com/gamercade-io/gamercade_site/issues/9
- [ ] #47 
- [ ] #71

**Comment by Andre-LA** [comment]
If possible add #9 to the list, otherwise making menus will be a bit harder to implement.

**Comment by RobDavenport** [comment]
Thanks for the suggestion @Andre-LA , unfortunately, there's still a lot of questions about how we want to handle this feature. For example, different font sizes for different resolutions, what exactly a "font" should be (bit depth, etc). The issue itself has more details about this though.

Currently it would be possible to do a workaround by using sprite images as your "font" and just calling `draw_sprite` for each letter. Not ideal, but I expect that once the text feature is implemented it will just be sugar for that.
